src/praisonai-agents/praisonaiagents/push/client.py

```
@@ -0,0 +1,388 @@
+                if msg_type == "_poll_empty":
+                    continue
+
+                # Auto-ACK for delivery guarantees (non-blocking)
+                event_id = data.get("event_id")
+                if event_id and msg_type not in (
+                    "ack_received", "nack_received", "channel.subscribed",
+                    "channel.unsubscribed", "channel.published", "channel.created",
+                    "channel.list", "presence.heartbeat_ack", "presence.list",
+                    "joined", "left", "error",
+                ):
+                    asyncio.create_task(self._send({"type": "message_ack", "event_id": event_id}))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)